README.md

```diff
@@ -82,23 +85,26 @@ def eval_fun(model: Any, genes: "Genes", shuffle_indices: Optional[List[int]] =
     return res
 ```
 
-3) Configure the FASTENER
+3. Configure the FASTENER
 
 By default fastener runs for 1000 iterations. The number of iterations can be adjusted with `number_of_rounds` parameter in the `fastener.Config()`.
 
 ```python
```

Comment:
Adhere to pep8 standards in examples: 
- Use whitespace around operators: `a = 3`
- Comments should start with "# "
- There is no whitespace around keyword arguments, except when types are specified